src/controllers/utils/advanced-search-validation.ts

```diff
@@ -115,9 +115,9 @@ function isDateInFuture (date: string) : boolean {
 }
 
 function isSicCodeFormatCorrect (value: string): boolean {
```

Comment:
this method was removed in master as it was no longer required - doing a pull should resolve this

Reply:
merged, should be sorted now

Reply:
Cool, approved ✅